app/src/main/java/io/btkelly/gnagtest/MainActivity.java

```diff
@@ -1,13 +1,20 @@
 package io.btkelly.gnagtest;
 
-import android.support.v7.app.AppCompatActivity;
+import android.app.Activity;
 import android.os.Bundle;
+import android.util.Log;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends Activity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        try{
+            Log.e("TEST", "Should fail Gnag Check");
+        } catch (Exception e) {
```

Comment:
<b>Reporter: </b>PMD<br /><b>Violation: </b><a target="_blank" href="https://pmd.github.io/pmd-5.4.2/pmd-java/rules/java/strictexception.html#AvoidCatchingGenericException">AvoidCatchingGenericException</a><br /><b>Notes: </b>Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block